lib/vttcue.js

```diff
@@ -258,5 +304,5 @@
     return WebVTT.convertCueToDOMTree(window, this.text);
   };
 
-  root.VTTCue = root.VTTCue || VTTCue;
+  root.VTTCue = VTTCue || root.VTTCue;
```

Comment:
Not in any reliable way.. You could check to see if properties like `lineAlign` exist. There in Gecko, but not in Chrome as of right now.